templates/Reference.md

```diff
@@ -94,6 +97,7 @@ Various commands are provided for reading and interpreting multiple bytes.
 | ascii *len* *label* | Reads *len* bytes as ASCII | `ascii 32 "Name"` |
 | utf16 *len* *label* | Reads *len* bytes as UTF16 (via current endian) | `utf16 12 "Name"` |
 | str *len* *encoding* *label* | Reads *len* bytes using the specified *encoding* identifier [v2.11+] | `str 8 "utf8" "Name"` |
+| cstr *encoding* *label* | Reads a sequence of zero-terminated bytes using the specified *encoding* identifier | `cstr "utf8" "Name"` |
```

Comment:
I was just going to create a PR adding the same documentation when I found this one. The only difference to my changes is the wording here:
```suggestion
| cstr *encoding* *label* | Reads a sequence of null-terminated bytes using the specified *encoding* identifier | `cstr "utf8" "Name"` |
```

To my knowledge *null-terminated* is the most commonly used terminology for this way of encoding strings. See for example:
* https://en.wikipedia.org/wiki/C_string_handling
* https://en.wikipedia.org/wiki/Null-terminated_string